src/inputs/getDesiredName.ts

```diff
@@ -12,7 +12,6 @@ export async function getDesiredName(multiStep: IMultiStepData): Promise<string>
     return await new Promise<string>((resolve, reject) => {
       const input = window.createInputBox();
       input.step = multiStep.step;
-      input.totalSteps = multiStep.totalSteps;
```

Comment:
We no longer will know ahead of time how many total steps there will be so removing this for now -- in the future we could add it back after the first step is selected since we would know after that.